### ISSUE TITLE
Feature request: applying filters to wekans by appending a query string to the url

### ISSUE THREAD

**Issue by thrademaker** [wekan/wekan#4540]
Dear all,
I would appreciate a feature which allows you to pass on filters, e.g. by appending a query string for the filter for "assignee" "JohnDoe" to the board's url : wekan.mycompany.com/b/ABDCefG123/my-project-kanban?assignee=johndoe
- FYI - use case scenario:
* I like embedding Wekans in SharePoint pages of a project using <iframe>
* I would like to create a sub page for a team with an embedded filtered Wekan on top 
Thank you all!

**Comment by xet7** [comment]
@thrademaker 

Does SharePoint even allow embedding WeKan at all? Have you tried? It looks like from this that there could problems:
https://www.sharepointed.com/2019/12/how-to-use-an-iframe-in-a-modern-sharepoint-online-page/

Any funding to https://wekan.team/commercial-support/ ?

**Comment by thrademaker** [comment]
@xet7 

Yes, my company just upgraded SP and I have used the embed webpart after updating HTML field security with our wekan server address and la voilà. I can use the embedded wekan. I have not yet been using this for long, but I do quite like the experience. 

I think would come handy to filter the embedded wekan e.g. to a certain team and have their according document library beneath so the team could focus on their tasks and their files. I will check with our admins, if I was just lucky getting the embed to work and will discuss commercial support, too.

**Comment by xet7** [comment]
Thanks!

**Comment by xet7** [comment]
Related to Multiverse Router.
